XU-TechnicalPaper.md

```diff
@@ -25,163 +25,176 @@
   4.2. [Blockstream's Liquid](#42-blockstreams-liquid)  
   4.3. [Lightning/Raiden](#43-lightning--raiden)  
   4.4. [Alphapoints Remarketer](#44-alphapoints-remarketer)  
-  4.5. [Altcoin.io/Etherdelta/Airswap/0x](#45-altcoinio--etherdelta--airswap--0x)  
-  4.6. [RaidEX.io](#46-raidexio)  
-5. [Requirements](#5-requirements-overview)
+  4.5. [Altcoin.io/Etherdelta/Airswap/0x](#45-altcoinio--etherdelta--airswap--0x--idex)  
+  4.6. [sparkswap](#46-sparkswap)  
+  4.7. [RaidEX.io](#47-raidexio)  
+  4.8. [Waves](#48-waves) 
+
 
 ## 1. Executive Summary
 
 1.1. The Vision
 ---------------
-Exchange Union (XU) connects digital asset exchanges by forming **a decentralized network**, which enables **instant and trustless** trades between digital asset exchanges. It enables cross-exchange trading for new trading pairs, access to the best price, and increased liquidity. Read the initial [White Paper](https://api.exchangeunion.com/pdf/en/ExchangeUnion-WhitePaper.pdf) which describes the idea and its use cases.
+Exchange Union (XU) connects digital asset exchanges by forming **a decentralized network**, which enables **instant and trustless** trades between digital asset exchanges, as well as individual traders. It enables cross-exchange trading for increased liquidity, access to the best price and an additional revenue stream for liquidity providers.
 
 1.2. Project Intro
 ------------------
-This specification describes and summarizes commonly agreed technical requirements for Exchange Union and thus should be referred to as “the bible” ;). Exchange Union’s implementation is challenging and can only be achieved with support of a greater developer community. The decision was made to open-source all documentation and the entire code base and to prioritize building a developer community, while also engaging full-time open-source developers to contribute to the development effort. Strategic partnerships with relevant companies will be formed. The business model for funding development is based entirely on Exchange Union's XUC token, which is already in circulation and doesn’t require an ICO.
+The Exchange Union project is powered by [XUD](https://github.com/ExchangeUnion/xud/), short for **Exchange Union Daemon**, the  open source node software which is operated by exchanges (or traders) and comprises the network. This document describes the inner workings of XUD from a product & technical perspective and summarizes technical requirements. For an overview of idea, use case & benefits read the [White Paper](https://www.exchangeunion.com/static/files/ExchangeUnion-WhitePaper_en.pdf), for hands-on guides check our [Wiki](https://github.com/ExchangeUnion/xud/wiki) and for code-level documentation our [TypeDoc](https://exchangeunion.github.io/xud-typedoc/). Chat with us on [Gitter](https://gitter.im/exchangeunion/Lobby) and follow major event & development milestones on our [blog](https://blog.exchangeunion.com/).
```

Comment:
Maybe start the third sentence "For an overview of the project read the White Paper..." because having a comma-separated list within a comma-separated list can be confusing.

Reply:
Fixed